File: guanabara-java/src/com/danillo/algoritmos/lista03/Exercicio37.java
package com.danillo.algoritmos.lista03;

import java.util.Scanner;

public class Exercicio37 {
    public static void main(String[] args) {
        Scanner entrada = new Scanner(System.in);

        System.out.println("Insira o salário atual: ");
        float salario = entrada.nextFloat();
        System.out.println("Insira o gênero (M ou F): ");
        char genero = entrada.next().charAt(0);
        System.out.println("Insira o tempo de serviço na empresa");
        int tempoServicoEmpresa = entrada.nextInt();

        float novoSalario;

        if(genero == 'F'){
            if(tempoServicoEmpresa < 15){
                novoSalario = salario + (salario * 5/100);
            } else if (tempoServicoEmpresa <=20) {
                novoSalario = salario + (salario * 12/100);
            } else{
                novoSalario = salario + (salario * 23/100);
            }
        }else{
            if(tempoServicoEmpresa < 20){
                novoSalario = salario + (salario * 3/100);
            } else if (tempoServicoEmpresa <=30) {
                novoSalario = salario + (salario * 13/100);
            } else{
                novoSalario = salario + (salario * 25/100);
            }
        }

        System.out.println("Salário reajustado: R$ " +novoSalario);
    }
}

File: guanabara-python/src/lista03/exercicio37.py
salario_atual = float(input("Insira o salário do funcionário: "))
genero = input("Insira o gênero do funcionário (M ou F): ")
tempo_empresa = int(input("Insira o tempo de serviço do funcionário na empresa: "))

if genero == 'F':
    if tempo_empresa < 15:
        aumento = (salario_atual * 5)/100
    elif tempo_empresa >= 15 and tempo_empresa >= 20:
        aumento = (salario_atual * 12) / 100
    else:
        aumento = (salario_atual * 23) / 100
else:
    if tempo_empresa < 20:
        aumento = (salario_atual * 3) / 100
    elif tempo_empresa >= 20 and tempo_empresa >= 30:
        aumento = (salario_atual * 13) / 100
    else:
        aumento = (salario_atual * 25) / 100

novo_salario = salario_atual + aumento
print(f"Salário reajustado: R$ {novo_salario}")
